ipc/provider/src/manager/evm/manager.rs

```diff
@@ -1020,6 +1041,29 @@ impl EthSubnetManager {
         }
     }
 
+    pub async fn handle_txn_token<B, D, M>(
```

Comment:
I think I am missing the point of this function. Could you please explain?

Reply:
because different combination of collateral source and supply source will have different txn value, if that's the question?

Reply:
Alright I get it now. It is a bit unclear at first sight. Maybe comment would do.

Reply:
comment added!